contrib/win32/win32compat/termio.c

```diff
@@ -103,12 +103,11 @@ ReadThread(_In_ LPVOID lpParameter)
 
 				if (!SetConsoleMode(GetStdHandle(STD_INPUT_HANDLE), dwAttributes))
 					error("SetConsoleMode on STD_INPUT_HANDLE failed with %d\n", GetLastError());
-			}
+			}			
```

Comment:
revert this unwanted change

Reply:
This is  out code under contrib/win32/win32compat/termio.c. Those white space are not necessary, so remove them.

Reply:
done